dali/operators/geometry/mt_transform_attr.cc

```diff
@@ -127,11 +127,11 @@ void MTTransformAttr::ProcessMatrixArg(const OpSpec &spec, const ArgumentWorkspa
     if (static_cast<int>(mtx_.size()) != output_pt_dim_ * input_pt_dim_) {
       mtx_.resize(output_pt_dim_ * input_pt_dim_, 0);
       MakeDiagonalMatrix(mtx_, 1);
-      Repeat(per_sample_mtx_, mtx_, N);
-      if (is_fused) {
-        translation_.resize(output_pt_dim_, 0);
-        Repeat(per_sample_translation_, translation_, N);
-      }
+    }
+    Repeat(per_sample_mtx_, mtx_, N);
+    if (is_fused) {
+      translation_.resize(output_pt_dim_, 0);
+      Repeat(per_sample_translation_, translation_, N);
```

Comment:
Won't this always be zero-filled?
```suggestion
      per_sample_translation_.resize(N * output_pt_dim_, 0);
```

Reply:
Done.